syft/pkg/cataloger/common/generic_cataloger_test.go

```diff
@@ -51,7 +51,7 @@ func TestGenericCataloger(t *testing.T) {
 	}
 
 	if len(actualPkgs) != len(expectedPkgs) {
-		t.Fatalf("unexpected packages len: %d", len(actualPkgs))
+		t.Fatalf("unexpected packages len: %d != %d", len(expectedPkgs), len(actualPkgs))
```

Comment:
I'd suggest changing this to use an `assert` function from testify to handle this. That should also make it more clear to the consumer of the test output which value is which.

Reply:
Pulled in `assert.Len` to address this 👍